pyspecProcScripts/correlation_alignment.py

```diff
@@ -114,14 +151,21 @@ def correl_align(s, align_phases=False,tol=1e-4,indirect_dim='indirect',fig_titl
                 fl.image(correl.C.setaxis('vd','#').set_units('vd','scan #'),human_units=False)
         correl.ft_clear_startpoints('t2')
         correl.ft('t2', shift=True, pad=2**14)
-        correl.ft(['DeltaPh1','DeltaPh2'])
-        correl = correl['DeltaPh1',ph1_selection]['DeltaPh2',ph2_selection] + correl['DeltaPh1',0]['DeltaPh2',0]
+        if ODNP:
+            correl.ft(['DeltaPh1'])
+            correl = correl['DeltaPh1',ph1_selection] + correl['DeltaPh1',0]
+        else:
+            correl.ft(['DeltaPh1','DeltaPh2'])
+            correl = correl['DeltaPh1',ph1_selection]['DeltaPh2',ph2_selection] + correl['DeltaPh1',0]['DeltaPh2',0]
```

Comment:
I'm confused by the addition here -- why are we adding the 0,0 element along the DeltaPh dimensions?  Someone needs to explain this to me in terms of the math in the paper.

Reply:
I spoke to Alec and asked and he said it was actually your idea and had something to do with adding in signal from the FID channel to just increase the signal? He wasn't quite sure either but had said it was your addition. should I just take it out?